src/Util/OperationTypes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Util;
+
+final class OperationTypes
```

Comment:
`ApiPlatform\Core\OperationType`, like https://github.com/giggsey/libphonenumber-for-php/blob/master/src/PhoneNumberType.php